tests/Microsoft.Identity.Web.Test.LabInfrastructure/Microsoft.Identity.Web.Test.LabInfrastructure.csproj

```diff
@@ -17,7 +17,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Identity.Client" Version="4.16.1" />
```

Comment:
@jennyf19 @pmaytak 
Given that the `Microsoft.Identity.Web.Test.LabInfrastructure.csproj` project references `Microsoft.Identity.Web.Test.Common.csproj` which itself [includes](https://github.com/AzureAD/microsoft-identity-web/blob/25ff0df874662d92225ccef28ae60c50ce5406c5/tests/Microsoft.Identity.Web.Test.Common/Microsoft.Identity.Web.Test.Common.csproj#L26) the Microsoft.Identity.Web project, MSAL.NET is provided by transitivity, and therefore this package reference is no longer necessary

Reply:
ok. i've tried taking it out before and that hasn't worked, but i'll try again. thanks for the suggestion.